app/packs/src/apps/mydb/elements/details/samples/SampleDetails.js

```diff
@@ -1499,10 +1499,8 @@ export default class SampleDetails extends React.Component {
       && this.state.activeTab) || visible.get(0);
 
     return (
-      <Card
-        className="eln-panel-detail"
-      >
-        <Card.Header className={"text-bg-" + (sample.isPendingToSave || isChemicalEdited ? 'info' : 'primary')}>
+      <Card className={"detail-card" + ((sample.isPendingToSave || isChemicalEdited) ? " detail-card--unsaved" : "")}>
```

Comment:
Maybe consider using `classnames` here?
```js
classnames(
    'detail-card',
    {'detail-card--unsaved': sample.isPendingToSave || isChemicalEdited}
)
```

Reply:
I'm still on the fence about classnames. If we should come to the conclusion, that it's use justifies the dependency I would roll it out throughout the entire codebase. For now I simplified the condition outside the return function.